src/makeArmy.js

```diff
@@ -20,8 +20,24 @@
  *
  * @return {function[]}
  */
+const makeShooters = function(serialNum) {
+  return () => serialNum;
+};
+
 function makeArmy(count) {
   // write code here
+  const army = [];
+  let shooterNum = 0;
+
+  while (shooterNum < count) {
+    const shooter = makeShooters(shooterNum);
+
+    army[shooterNum] = shooter;
+
+    shooterNum++;
+  }
```

Comment:
```suggestion
for(let i=0; i<count; i++) {
  army.push(() => i)
}
```
Do you need separate function?